packages/parser-typescript/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "esModuleInterop": true,
+        "outDir": "dist",
+        "strict": true
+    },
+    "exclude": [
+        "dist",
+        "node_modules"
+    ],
+    "extends": "../../tsconfig.json",
+    "include": [
+        "src/**/*.ts",
+        "tests/**/*.ts"
+    ],
+    "references": [
```

Comment:
Do we need to update the global references? We tend to forget them but I'm not sure when they are needed.
Maybe we should add a step in `create-hint/parser` packages that if it's in the official package it automatically runs the update dependencies script?

Reply:
Yep, forgot to do that. Just pushed a commit with the update - I'll rebase and splice it in once we're happy with the rest of the PR.

To make `update:dependencies` work correctly we need to re-run it each time our dependencies in `package.json` change too. Maybe we should just make this part of `build` or `test` for all our packages.

Reply:
I'm happy with the rest of the PR and @sarvaje as well. If you want to go ahead and merge feel free!

Reply:
Ok, I'm going to rebase on `master` and splice in that last commit. Once CI passes after that update I'll go ahead and merge.